tes-adapter/src/main/java/com/epam/pipeline/Application.java

```diff
@@ -0,0 +1,11 @@
+package com.epam.pipeline;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+@SpringBootApplication
+public class Application {
```

Comment:
What do you think about introducing WebConfiguration class, at least for enable `@EnableWebMvc` there, don't forget to `Import` it here